File: csharp/ButtplugCSharpFFI/ButtplugPingException.cs
using System;

namespace Buttplug
{
    public class ButtplugPingException : ButtplugException
    {
        /// <inheritdoc />
        public ButtplugPingException()
        {
        }

        /// <inheritdoc />
        public ButtplugPingException(string aMessage) : base(aMessage)
        {
        }

        /// <inheritdoc />
        public ButtplugPingException(string aMessage, Exception aInner) : base(aMessage, aInner)
        {
        }
    }
}

File: java/lib/src/main/java/io/buttplug/ButtplugPingException.java
package io.buttplug;

public class ButtplugPingException extends ButtplugException {
    public ButtplugPingException(String msg) {
        super(msg);
    }

    public ButtplugPingException(String msg, Exception inner) {
        super(msg, inner);
    }
}
